client-lite/src/exe/docs.cpp

```diff
@@ -139,8 +139,11 @@ HRESULT Run() try
 
     RestPortAdvertiser portAdvertiser(controller.Port());
     DoLogInfo("Port number written to %s", portAdvertiser.OutFilePath().data());
+    
+    #ifndef UBUNTU_CORE_SNAP_ONLY
```

Comment:
Directives are always placed at the left margin, so reduce indentation by one level for all 3 lines here.